src/demo/__init__.py

```diff
@@ -8,3 +8,4 @@
     predict_MerchantSize_on_lead_data_demo,
     preprocessing_demo,
 )
+from .pipeline_utils import *
```

Comment:
Not sure about that, but shouldn't we also do `from .console_utils import *` and `from .demos import *`

Reply:
It is not a "must" but "pratical". Addressed.